community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/OffHeapPageLock.java

```diff
@@ -184,7 +184,7 @@ public static boolean tryWriteLock( long address )
             boolean writeCountOverflow = (s & CNT_MASK) == CNT_MASK;
 
             // bitwise-OR to reduce branching and allow more ILP
-            if ( unwritablyLocked | writeCountOverflow )
```

Comment:
Did you see the comment here?

Reply:
Sorry, missed that one. I can revert it after benchmarks finish.